conf/Autorun.config

```diff
@@ -15,16 +15,35 @@ profiles {
     }
   }
 
-  // Profile with parameters for runs using the Human_SG bams as input.
-  SG {
-   params{
+  // A profile with all the local paths to required files. 
+  // These will need to be provided manually by anyone wanting to reproduce the results outside of the EVA filesystem.
+  local_paths {
+    params {
       // Mapping reference and reference indexes
       // These are required by eager for Damage calculation etc. No mapping is taking place here.
-      fasta = '/mnt/archgen/Reference_Genomes/Human/hs37d5/hs37d5.fa'
-      fasta_index = '/mnt/archgen/Reference_Genomes/Human/hs37d5/hs37d5.fa.fai'
-      bwa_index = '/mnt/archgen/Reference_Genomes/Human/hs37d5/'
-      seq_dict = '/mnt/archgen/Reference_Genomes/Human/hs37d5/hs37d5.dict'
+      fasta                 = '/mnt/archgen/Reference_Genomes/Human/hs37d5/hs37d5.fa'
+      fasta_index           = '/mnt/archgen/Reference_Genomes/Human/hs37d5/hs37d5.fa.fai'
+      bwa_index             = '/mnt/archgen/Reference_Genomes/Human/hs37d5/'
+      seq_dict              = '/mnt/archgen/Reference_Genomes/Human/hs37d5/hs37d5.dict'
+      
+      // Qualimap bedfile for on-target coverage calculation
+      snpcapture_bed        = '/mnt/archgen/Reference_Genomes/Human/hs37d5/SNPCapBEDs/1240K.pos.list_hs37d5.0based.bed'
```

Comment:
Warning: this will mean you won't get whole genome coverages and other statistics, I would assume this is only applicable to TF data (I would assume).

It might be good to set this to `null` in non 1240K profiles.

Reply:
This is added here so that the on-target and off-target can be compared to assess capture efficiency. By having only on-target in both cases the results are directly comparable. I suppose SG could be across the genome, as we expect no difference, but it is not technically what has been discussed in BigData meetings.